xp/xconnect/xconnect-xp0.json

```diff
@@ -0,0 +1,641 @@
+// -------------------------------------------------------------------------- //
```

Comment:
deze moet niet in git

Reply:
Nee ik moest de json iets aanpassen. Is nu niet meer relevant omdat die opzet niet werkte.